VGMPlay/xdg/vgmplay-mime.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
+    <mime-type type="audio/x-vgm">
+        <comment>VGM Audio File</comment>
```

Comment:
Personally I would use an application MIME-type as VGM files don't really encode audio (outside datablocks)

Reply:
I used it because it already exists. Some application I installed on wine set it in the past, and also gstreamer's gmedec uses it. https://gstreamer.freedesktop.org/data/doc/gstreamer/head/gst-plugins-bad-plugins/html/gst-plugins-bad-plugins-gmedec.html